pkg/kube/client.go

```diff
@@ -339,22 +353,25 @@ type client struct {
 	extInformer kubeExtInformers.SharedInformerFactory
 
 	kube         kubernetes.Interface
-	kubeInformer informers.SharedInformerFactory
+	kubeInformer xnskubeinformer.SharedInformerFactory
 
 	dynamic         dynamic.Interface
-	dynamicInformer dynamicinformer.DynamicSharedInformerFactory
+	dynamicInformer xnsinformers.DynamicSharedInformerFactory
 
 	metadata         metadata.Interface
 	metadataInformer metadatainformer.SharedInformerFactory
 
 	istio         istioclient.Interface
-	istioInformer istioinformer.SharedInformerFactory
+	istioInformer xnsistioinformer.SharedInformerFactory
 
 	gatewayapi         gatewayapiclient.Interface
-	gatewayapiInformer gatewayapiinformer.SharedInformerFactory
+	gatewayapiInformer xnsgatewayapiinformer.SharedInformerFactory
 
-	started atomic.Bool
 	// If enabled, will wait for cache syncs with extremely short delay. This should be used only for tests
```

Comment:
Is this comment duplicate of [line 374](https://github.com/maistra/istio/pull/814/files#diff-137bba79988de442c1fde0fc29520d6d7e5a5239ceac82f5439a5bf513ebb2d4R374)?

Reply:
Yes, look like a mistake. I removed the added comment.